MAUI/Autocomplete/Selection.md

```diff
@@ -141,6 +141,56 @@ The selected items can be changed programmatically by using the [SelectedItems](
 
 {% endtabs %}
 
+### TokensWrapMode
+
+There are two ways to display multi selection items in the AutoComplete control. They are
+
+* Wrap
+* None
+
+#### Wrap mode
+
+When the TokensWrapMode is set to Wrap the selected items will be wrapped to the next line of the SfAutoComplete.
```

Comment:
When the TokensWrapMode is set to Wrap, the selected items will be wrapped to the next line of the SfAutoComplete.